### DISCUSSION TITLE
do u play genshin impact

### DISCUSSION THREAD

**Discussion by chitang233** [KimmyXYC/clash-rules#2] [Q&A] [answered]
玩游戏不玩原神，就像沙盒不玩迷你世界，追星不追肖战，读大学不读宁波工程学院，打篮球不看落山鸡糊人，生活区不看lex，学跳舞不看党妹，读书不读唐家三少，看新闻不看观察者网，moba不玩王者荣耀，看日漫不看我的英雄学院，vtb不看桐生可可，哔哩哔哩不关注陈睿，说明这个人文学造诣和自我修养不足如果你早上吃饭香甜，那一定有米桑的艰苦付出。 如果你晚上大便不通，那必定是腾讯在从中作梗。他理解不了这种内在的阳春白雪的高雅艺术，他只能看到外表的辞藻堆砌，参不透其中深奥的精神内核，他整个人的层次就卡在这里了，只能度过一个相对失败的人生

**Comment by KimmyXYC** [comment, marked as answer]
前进道路上，全党要始终牢记全心全意为人民服务的根本宗旨，坚持一切为了人民，一切依靠人民，时刻与人民想在一起、干在一起，始终赢得人民的拥护和支持，聚合起十四亿多中国人民的磅礴之力，众志成城推进中华民族伟大复兴历史进程，奋力谱写全面建设社会主义现代化国家崭新篇章。
